examples/server-example.py

```diff
@@ -74,11 +72,26 @@ def multiply(parent, x, y):
     uri = "http://examples.freeopcua.github.io"
     idx = server.register_namespace(uri)
 
-    # get Objects node, this is where we should put our custom stuff
+
+    # create a new node type we can instantiate in our address space
+    base_type = server.get_root_node().get_child(["0:Types", "0:ObjectTypes", "0:BaseObjectType"])
```

Comment:
It looks good. It is also applicable if we ever create nodes as python objects, than you would be able to do this on any node/type :)

Great idea guys!